papaparse.js

```diff
@@ -1285,21 +1285,25 @@
 							continue;
 						}
 
-						// Closing quote followed by delimiter
-						if (input[quoteSearch+1] === delim)
+						var spacesBetweenQuoteAndDelimiter = extraSpacesLengthAfterQuote(nextDelim);
+
+						// Closing quote followed by delimiter or 'unnecessary steps + delimiter'
+						if (input[quoteSearch+1] === delim  || spacesBetweenQuoteAndDelimiter > 0)
```

Comment:
Why not summing the spaces to the QuoteSearch?

Reply:
Good point.